README.md

```diff
@@ -1 +1,40 @@
-# portfoliolive
\ No newline at end of file
+
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Portfolio-Desktop-Version
+
+> In this project, I set up my portfolio for the Desktop. This project includes a complete replica of a porfolio-template from Figma. It is only the front-end project so it only contains .html and .css files.
+
+> Link to my portfolio: https://wahaj-ali.github.io/portfolio.github.io/
```

Comment:
- [ ] It's better if you follow the [template](https://github.com/microverseinc/readme-template) of the README file provided by Microverse. There is a special heading for the live demo which makes it more visible and professional ✨